OneCommunityGlobal/HGNRest
# Add new field "Start Date" to Create New User modal

# Description
See [PR1126](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1126)

## Related PRS (if any):
To test this backend PR you need to checkout [PR1126](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1126) in the frontend.

## Main changes explained:
- createdDate field is now set from value in front-end 
- createdDate is defaulted to the next day
- added new field `isFirstTimelog` to the userProfile model to determine the user's first timelog (boolean set to true)

## How to test:
1. check into current branch and front branch with the same name
2. do `npm run build` and `npm start` to run this PR locally
3. follow steps in the front-end
